### DISCUSSION TITLE
ManageBac

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2252] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

AFAIK, There's no easy way to know except to inspect the site. But I'm ready to help!

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by isabelroses** [comment]
Seeing as this is website, this could be achieved as a userstyle. For anyone willing to create the theme checkout https://github.com/catppuccin/userstyles.
